GLIB uses the technique of defining the macro to nothing
if it can't be used on a given compiler. But having this
defined to nothing generates a Werror, so that can't work.

--- meson.build.orig	2022-11-02 00:37:51.000000000 -0700
+++ meson.build	2022-11-02 00:38:13.000000000 -0700
@@ -306,7 +306,6 @@
     'int-to-pointer-cast',
     'main',
     'missing-braces',
-    'missing-declarations',
     'missing-prototypes',
     'nonnull',
     'pointer-to-int-cast',
